src/components/ui/Figure.astro

```diff
@@ -26,7 +25,7 @@ interface Props extends VariantProps<typeof figureVariants> {
 const { size, class: className, ...props } = Astro.props;
 ---
 
-<figure class={clsx(figureVariants({ size, className }))}>
+<figure class={figureVariants({ size, className })}>
```

Comment:
Shouldn't this be `class:list`?

Reply:
I don't think so because we only pass one value to the class. If instead of passing `className` to `figureVariants` we wanted to pass it to the class, then yes `class:list` would be the way to go.